.github/workflows/on-demand-vrfv2plus-performance-test.yml

```diff
@@ -14,10 +10,19 @@ on:
         description: "Regex for tests to run"
         required: false
         default: "(TestVRFV2PlusPerformance)"
+      test_config_override_path:
+        description: Path to a test config file used to override the default test config
+        required: false
+        type: string               
       test_secrets_override_key:
         description: 'Key to run tests with custom test secrets'
         required: false
         type: string 
+      chainlink_version:
+        description: Chainlink image version to use
+        default: develop
+        required: true
```

Comment:
chainlink_version should not be required if VRF tests run against existing env.

Reply:
@iljapavlovs fixed in b7a186c663f288a2d82dabdbd9f946ff461391d0